src/virtual_dom/node/el.rs

```diff
@@ -106,8 +106,15 @@ impl<Ms> El<Ms> {
 
     // todo: Return El instead of Node here? (Same with from_html)
     /// Create elements from a markdown string.
+    /// Enables all additional markdown extensions, see #392
```

Comment:
Can we change it to something like `/// _Note:_ All additional markdown [extensions](https://docs.rs/pulldown-cmark/latest/pulldown_cmark/struct.Options.html) enabled.`

Why:
- I don't know if it's a good idea to add note `see #392` - you don't know if it's issue or PR and primarily there would be too many similar notes in the code once we start to write related PRs/links for all changes.
- Missing dot at the end.
- You don't know what extensions because there is neither comment about the used markdown library nor link to docs.

Reply:
Yes, that's definitely better. Didn't write too many documentation yet, so I'm always trying to improve.